ui/package.json

```diff
@@ -40,9 +40,10 @@
   },
   "devDependencies": {
     "@babel/plugin-proposal-object-rest-spread": "^7.5.5",
+    "@babel/plugin-transform-block-scoping": "^7.10.4",
```

Comment:
this was the missing dependency that caused `make static-dist` to fail. we can likely remove this after upgrading to Octane.